nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -77,6 +77,7 @@ class BtActionNode : public BT::ActionNodeBase
     if (getInput("server_name", remapped_action_name)) {
       action_name_ = remapped_action_name;
     }
+    getInput("is_global", is_global_);
```

Comment:
Question: should this be applied to non-action nodes or only the action nodes?

Reply:
Also, I think this is also missing the other nodes that have the initialization procedure within their implementations themselves (not just the action server base node). I think that's a key part of the ticket https://github.com/ros-navigation/navigation2/issues/5036 but I can understand just starting in one spot to get it right before doing all the others!